agent/agent.go

```diff
@@ -2150,12 +2150,23 @@ func (a *Agent) loadServices(conf *config.RuntimeConfig) error {
 			return fmt.Errorf("failed reading service file %q: %s", file, err)
 		}
 
+		// If the file ended up empty as a result of something like an OS crash, remove
+		// it for convenience and log about it.
+		if len(buf) == 0 {
+			a.logger.Printf("[WARN] Removing leftover empty service file %q", file)
+			if err := os.Remove(file); err != nil {
```

Comment:
I don't think we should attempt to delete the file, just warn and skip it. Leaving it around might help operators see what's going on by looking at timestamps, etc.